### ISSUE TITLE
Bug: `Talk` dialogs inside point and zoom instances aren't skipped

### ISSUE THREAD

**Issue by Jaksuhn** [NightmareXIV/TextAdvance#35]

**Comment by Limiana** [comment]
Yeah, I know that. But I don't know why and lack of easily accessible instances of it makes it a bit harder to debug

**Comment by Jaksuhn** [comment]
From testing it seems those don't have a `MouseClick` component at all. They're solely a `MouseDown` followed by a `MouseUp`

**Comment by Limiana** [comment]
I'm pretty sure this is fixed by now
